testcases/cloud_admin/modify_instance_type.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python
+#
+# Description: Test the euca-modify-instance-type command works properly.
+#
+# NOTE: This testcase DOES NOT work with credentials file (credpath and eucarc) it only
+#       allows password/config_data for the time being.
+#
+from eucaops import Eucaops
+from eutester.eutestcase import EutesterTestCase
+
+import re
+import random
+
+class instance_type():  # mimicing a struct, I think this is the preferred way in python
+
+    # shaon pointed out that I should be using Vm_Type instead of my own class
+    # but can't figure that out right now - 4.28.2015 mbacchi
+    def __init__(self, name=None, cpus=None, mem=None, disk=None):
+        self.name = name
+        self.cpus = cpus
+        self.mem = mem
+        self.disk = disk
+
+    def p(self):
+        print ("Name: " + self.name + " cpu: " + str(self.cpus) + " mem: "
```

Comment:
This is a good place to use format rather than a bunch of string concatenation: https://docs.python.org/2/library/stdtypes.html#str.format